dali/test/python/test_dali_variable_batch_size.py

```diff
@@ -627,33 +665,40 @@ def get_data(batch_size):
         obj_num = random.randint(1, 20)
         test_box_shape = [obj_num, 4]
         test_lables_shape = [obj_num, 1]
-        bboxes = [np.random.random(size=test_box_shape).astype(dtype=np.float32) for _ in range(batch_size)]
-        labels = [np.random.randint(0, 255, size=test_lables_shape, dtype=np.int32) for _ in range(batch_size)]
+        bboxes = [
+            np.random.random(size=test_box_shape).astype(dtype=np.float32)
+            for _ in range(batch_size)]
```

Comment:
I think that it reads better that an opening brace on a new line is matched with a closing brace on its own line:
```suggestion
        bboxes = [
            np.random.random(size=test_box_shape).astype(dtype=np.float32)
            for _ in range(batch_size)
        ]
```
TBH, I don't even know if the standard allows this.
This should be legal:
```suggestion
        bboxes = [np.random.random(size=test_box_shape).astype(dtype=np.float32)
                  for _ in range(batch_size)]
```
Anyway, these are just suggestions - I won't push it.

Reply:
Maybe it's weird personal preference: line break on opening of list comprehension leaves a whole line for the expression inside and highlights it nicely, while there is no similar reason for doing so with the closing bracket. Contrary, it weirdly separates the whole comprehension from the block of code it rests in. Anyway, here it perfectly fits with no break, so I reverted the change.